src/pocket/PocketEventBridgeWithLambdaTarget.ts

```diff
@@ -57,20 +60,24 @@ export class PocketEventBridgeWithLambdaTarget extends PocketVersionedLambda {
    * @private
    */
   private createEventBridgeRule(
-    lambda: ApplicationVersionedLambda
+    targetLambdas: ApplicationVersionedLambda[]
```

Comment:
also this right now only allows lambda - we should consider connecting a single rule with different targets like sqs, kinesis etc.